### PR TITLE
HTML: remove deprecated <style type='text/css'> attribute

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The `type` attribute [is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style#deprecated_attributes) on the `<style>` tag.
It defaults to `text/css`.

The PR removes remaining occurrences in the repo.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/1.5/using_firefox_1.5_caching/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ In this example:
 <HTML>
 <head>
 <title>Order query Firefox 1.5 Example</title>
-<style type="text/css">
+<style>
```
Comment (Josh-Cena) [inline files/en-us/mozilla/firefox/releases/1.5/using_firefox_1.5_caching/index.md:96]: I don't think we need to fix historical texts?

**Comment by OnkarRuikar** [comment]
Just to confirm, we are not changing release docs for lint issues as well right?

**Comment by Josh-Cena** [comment]
That I think should be discussed case-by-case. I can't think of a general rule on top of my head—for example, we definitely want to add curly braces or normalize indentation, but code style would be more tricky.

File: files/en-us/web/svg/tutorial/gradients/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Linear gradients change along a straight line. To insert one, you create a {{SVG
         <stop offset="50%" stop-color="black" stop-opacity="0"/>
         <stop offset="100%" stop-color="blue"/>
       </linearGradient>
-      <style type="text/css"><![CDATA[
+      <style><![CDATA[
```
Comment (teoli2003) [inline files/en-us/web/svg/tutorial/gradients/index.md:35]: I don't know SVG enough to know if this has an impact, but this is the SVG `<style>` element, not the HTML one. 

I'm ok with removing it in HTML, but I would like to know if this is pertinent in SVG too.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/svg/tutorial/gradients/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/svg/tutorial/gradients/index.md:35]: The [default value](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/style#attributes) is `text/css` so there won't be any change to the code samples.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**